.github/ISSUE_TEMPLATE/enhancement_request.yml

```
@@ -0,0 +1,44 @@
+name: Enhancement Request
+description: Request an enhancement and/or open a science discussion
+title: "Put brief (<65 char) title here!"
+labels: ["enhancement"]
+body:
+  - type: textarea
+    id: enhancement-description
+    attributes:
+      label: What is the feature/what would you like to discuss?
+      placeholder: ex. I'd like to entirely refactor chemistry and here's how.
+    validations:
+      required: true
+  - type: textarea
+    id: user-reference
+    attributes:
+      label: Is there anyone in particular you want to be part of this conversation?
+      description: Add the user(s) with @<username>
+      placeholder: ex. @confucius
+    validations:
+      required: false
+  - type: dropdown
+    id: answer-changing
+    attributes:
+      label: Will this change answers?
```

In some sense, if it changes answers it is not an enhancement but a bug fix or science change. An enhancement should include a way to have it off by default and turn it on via a new namelist variable.

---

That is an interesting point. Since a science change/discussion also uses this template, I'm leaning towards leaving this? But I could add some additional text in the description about this.

Thoughts @cacraigucar and @nusbaume ?

---

I guess we should be more clear about the information we really want here. Internal to CAM, an enhancement, including most science changes, should come with a default configuration that does not change answers for regression tests. This allows scientists and SEs to dissect the causes of various changes to simulations. Perhaps a better wording here is:
```suggestion
      label: Will this change regression-test answers?
```

---

added clarifying language